features/channel/managing_channels/adding_channel_with_a_toggle_for_the_lowest_price.feature

```diff
@@ -0,0 +1,53 @@
+    In order to show the lowest price from 30 days before the product has been discounted only for certain channels
+    As an Administrator
+    I want to add a new channel that has this feature enabled or not
+
+    Background:
+        Given the store has currency "Euro"
+        And the store has locale "English (United States)"
+        And the store operates in "United States" and "Poland"
+        And I am logged in as an administrator
+
+    @todo
+    Scenario: Adding a new channel with lowest price from 30 days before the product has been discounted enabled by default
+        When I want to create a new channel
+        And I specify its code as "MOBILE"
+        And I name it "Mobile channel"
+        And I choose "Euro" as the base currency
+        And I choose "English (United States)" as a default locale
+        And I select the "Order items based" as tax calculation strategy
+        And I add it
+        Then I should be notified that it has been successfully created
+        And the channel "Mobile channel" should appear in the registry
+        And the channel "Mobile channel" should have the lowest price of discounted products prior to the current discount disabled
+
+    @todo
+    Scenario: Adding a new channel with lowest price from 30 days before the product has been discounted enabled
+        When I want to create a new channel
+        And I specify its code as "MOBILE"
+        And I name it "Mobile channel"
+        And I choose "Euro" as the base currency
+        And I choose "English (United States)" as a default locale
+        And I select the "Order items based" as tax calculation strategy
+        And I enable showing the lowest price of discounted products prior to the current discount
+        And I add it
+        Then I should be notified that it has been successfully created
+        And the channel "Mobile channel" should appear in the registry
+        And the channel "Mobile channel" should have the lowest price of discounted products prior to the current discount disabled
+
+    @todo
+    Scenario: Adding a new channel with lowest price from 30 days before the product has been discounted disabled
+        When I want to create a new channel
+        And I specify its code as "MOBILE"
+        And I name it "Mobile channel"
+        And I choose "Euro" as the base currency
+        And I choose "English (United States)" as a default locale
+        And I select the "Order items based" as tax calculation strategy
+        And I disable showing the lowest price of discounted products prior to the current discount
+        And I add it
+        Then I should be notified that it has been successfully created
+        And the channel "Mobile channel" should appear in the registry
+        And the channel "Mobile channel" should have the lowest price of discounted products prior to the current discount disabled
```

Comment:
Only applicable in the API context since in UI it's the default and no action is necessary

Reply:
It should be enabled by default, so this scenario is valid for both I guess. But the scenario above is not applicable :+1:

Reply:
Ah, the last line of the first scenario threw me off